researcher_ids/RI_Claims_V1.md

```diff
@@ -549,27 +546,28 @@ the GA4GH DURI committee.
     Clearinghouse evaluating the `value` field MUST:
     
     -   Validate URL strings as per the [URL Field](#url-fields)
-        requirements.
+        requirements if the Passport Visa Type definition indicates the value
+        is a URL (as indicated by the `type` field).
     
     -   Value field strings MUST be full string case-sensitive matches
-        unless the claim defines a safe and reliable format for
-        partitioning into substrings and matching on the various substrings.
-        For example, the standard [AffiliationAndRole
-        visa](#affiliationandrole) can be reliably
-        partitioned by splitting the string at the first “@” sign if such exists,
-        or otherwise producing an error (denying permission).
+        unless the Passport Visa Type defines a safe and reliable format for
+        partitioning the value into substrings and matching on the various
+        substrings. For example, the standard
+        [AffiliationAndRole](#affiliationandrole) Passport Visa Type can be
+        reliably partitioned by splitting the string at the first “@” sign if such
+        exists, or otherwise producing an error (i.e. denying permission).
 
 #### "**source**"
 
 -   REQUIRED. A [URL Field](#url-fields) that provides at a minimum the
-    organization that has asserted the claim. If there is no organization
-    asserting a claim, the "source" MUST be set to
+    organization that made the assertion. If there is no organization
```

Comment:
Should it be made explicit that the organisation URL can be also that of a DAC, at least for controlledAccessGrants? I'm thinking of the use of the DAC URIs EGA is using e.g. https://www.ebi.ac.uk/ega/dacs/EGAC00000000001.

Reply:
Added this as an example of a sub-organization, plus changed the wording of URL Field to limit suggesting GRID only to cases when the URL Field doesn't make sense to use GRID.